concepts/cond/about.md

```diff
@@ -1,14 +1,14 @@
-When we want to have branching code, we can use the [`cond/1`][cond] function:
```

Comment:
`cond` is a macro, but I think it's to early to explain the difference, so it's easier to refer to it as just "cond".